library/core/src/result.rs

```diff
@@ -1113,11 +1113,15 @@ impl<T, E> Result<T, E> {
         }
     }
 
-    /// Returns the contained [`Ok`] value or a default
+    /// Consumes the `self` argument and returns the contained [`Ok`] value
+    /// or the default value.
```

Comment:
Let's keep consistency with `unwrap` and other methods:

```suggestion
    /// Returns the contained [`Ok`] value or the default value,
    /// consuming the `self` value.
```